docker/etc/s6/openssh/setup

```diff
@@ -24,6 +24,10 @@ if [ ! -f /data/ssh/ssh_host_ecdsa_key ]; then
     ssh-keygen -t ecdsa -b 256 -f /data/ssh/ssh_host_ecdsa_key -N "" > /dev/null
 fi
 
+if [ "${GITEA_CUSTOM}" != "/data/gitea" ]; then
+    sed -i "s#/data/gitea#${GITEA_CUSTOM}#" /etc/profile.d/gitea.sh
```

Comment:
I would be better to find the line based on `GITEA_CUSTOM=.*` since if GITEA_CUSTOM already has been changed previously it will not override it.

Reply:
See my comment in the PR body :)

> Environment changes trigger a recreation of the container and reset the profile script in /etc/profile.d/gitea.sh. It is safe to replace the default path instead of the full line.

Reply:
Oh just misread. Simply delete the /etc/profile.d/gitea.sh as it a env variable declared in dockerfile. This will not be needed.

Reply:
Done in 7705d4d43441b08bd7cdb558088cfa0956eb05d8.